### ISSUE TITLE
<tts> tag ignored before <hr id=answer> without use of {{Front}}

### ISSUE THREAD

**Issue by ShaddyDC** [ankidroid/Anki-Android#5116]
###### Reproduction Steps

1. Create a new Card type
2. Use Back Template like this

```
<tts service="android" voice="en_GB">This will not be read</tts>
<hr id=answer>
This may be read
```

###### Expected Result

No language prompt and "This will not be read" to be read out when showing answer

###### Actual Result

1. A prompt for the language to be used in tts when showing answer
2. It reads out "This may be read"

###### Debug info
AnkiDroid Version = 2.9alpha51

Android Version = 8.1.0

ACRA UUID = 63c3cd64-4bd0-4ad3-8dd4-980b14380bb9

###### Research
*Enter an [ x ] character to confirm the points below:*

[ x ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[ x ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[ x ] I have searched for similar existing issues here and on the user forum

###### Other

Just in case this sounds like an unreasonable edge-case: I adapted a copy of the "Basic and reversed" card to always have an expression at the top of the answer and the definition below. I wanted only the expression to be read out, and only after showing the answer. Thus I changed both backsides to something like this:

```
<tts service="android" voice="en_GB">{{Expression}}</tts>
<hr id=answer>
{{Definition}}
```

Switching to this fixed the tts behaviour

```
<tts service="android" voice="en_GB">{{Expression}}</tts>
<hr>
{{Definition}}
```

After having found the issue, this behaviour makes complete sense. You've usually only got `{{Front}}` before that line, and you don't want to read it out twice. I'm not sure if this behaviour is wrong or needs fixing. It is a misuse of `<hr id=answer>` on my part that is arguably wrong. I don't think this scenario is completely outlandish though, so adding  a disclaimer to the [docs](https://github.com/ankidroid/ankidroiddocs/blob/v2.9-preview/manual.asc#advanced) seems like a good "fix" to keep others from repeating my mistakes. 

On that note, for an actual bug, it seems commenting out the line has no effect, and this disables the tts tags as well:

```
<tts service="android" voice="en_GB">{{Expression}}</tts>
<!--<hr id=answer>-->
{{Definition}}
```

Also really minor, but maybe worth knowing about.

**Comment by juliuscrn** [comment]
Doesn't look like there's anything special about {{Front}}. My cards don't use {{Front}} at all and I am having the same issue. Unless I use <hr> after the field that I want to be read out, it will read out the entire card.

**Comment by david-allison** [comment]
We're going to be deprecating `<tts>` in favour of Anki Desktop's `{{tts}}`

This is fixed in the new code path, so closing this as 'won't fix'
